src/main/java/pl/edu/pw/elka/paprykaisalami/geeruh/issues/adapters/persistent/IssuePersistent.java

```diff
@@ -131,7 +135,7 @@ public static IssuePersistent of(Issue issue, StatusPersistent status, UserPersi
     @AllArgsConstructor
     @NoArgsConstructor
     @Data
-    static class IssuePersistentId implements Serializable {
+    public static class IssuePersistentId implements Serializable {
 
         String projectCode;
```

Comment:
Your implementation is perfectly fine, but I just thought that the comment might be more of an Issue's attribute, so maybe there could be a bidirectional mapping where you could have `issue.getComments()`, `issue.addComment()` and you then that could be even part of issue's response, but that's only a thought. It is probably unneeded, especially with your implementation.

Reply:
Yeah, I also thought of that, but decided that its less "coupled" this way. I'm not sure now which way is better, but I would keep this one seeing that the deadline is so near